src/App.tsx

```diff
@@ -3,14 +3,11 @@ import { JSONLoader } from './components/JSONLoader'
 import { JSONInspector } from './components/JSONInspector'
 import { JSONExplorer } from './components/JSONExplorer'
 import { ReactorEditor } from './components/ReactorEditor'
-import { useUpdateMonacoTsTypes } from './logic/useUpdateMonacoTsTypes'
 
 function App() {
   const [json, setJson] = useState<any>()
   const [processedJson, setProcessedJson] = useState<any>()
 
-  useUpdateMonacoTsTypes(json)
```

Comment:
Move this to the child component because I'd like to extract type from this hook and show them in future.